services/core-api/app/date_time_helper.py

```diff
@@ -1,5 +1,8 @@
 from dateutil.relativedelta import relativedelta
+from datetime import datetime
 
+def get_formatted_current_time():
+    return datetime.now().strftime("%d/%b/%Y %H:%M:%S")
```

Comment:
Minor, but I remember writing some datetime helper functions... They're in services/core-apii/app/api/utils/helpers.py. There aren't any methods that I saw that this duplicates, but it looks like it could be combined maybe. This is a pre-existing thing so feel free to ignore it!